src/cnv/ugbio_cnv/convert_combined_cnv_results_to_vcf.py

```diff
@@ -0,0 +1,183 @@
+import argparse
+import logging
+import statistics
+import subprocess
+import sys
+import warnings
+from os.path import join as pjoin
+
+import pandas as pd
+import pysam
+from ugbio_core.logger import logger
+
+warnings.filterwarnings("ignore")
+
+
+def add_vcf_header(sample_name, fasta_index_file):
+    header = pysam.VariantHeader()
+
+    # Add meta-information to the header
+    header.add_meta("fileformat", value="VCFv4.2")
```

Comment:
This function is completely(?) duplicated with the one in `convert_cnv_results_to_vcf.py`

Reply:
yes! convert_cnv_results_to_vcf.py is for cn.mops results only. 
made use of this function within that script as well.
also refactored convert_cnv_results_to_vcf.py and splited main to functions.